src/index.ts

```diff
@@ -71,6 +71,7 @@ export { AlertLevel } from './shared/alert/alert-level.model';
 export { Jurisdiction } from './shared/domain/definition/jurisdiction.model';
 export { RouterHelperService } from './shared/utils/router-helper.service';
 export { CaseType } from './shared/domain/definition/case-type.model';
+export { CaseTypeLite } from './shared/domain/definition/case-type-lite.model';
```

Comment:
my only question is why we have CCD domain model classes into the toolkit rather that management web. More a general concern rather than an issue specific with this PR